db/seeds.rb

```diff
@@ -18,56 +26,54 @@
     paused: true,
     }
 )
+u.save
+
+# Hold on to reference to the game
+game = u.game
 
 #An example message for the database. Probably doesn't need to exist in the final version.
-m = Message.create(
-	sender: "UK",
-	recipient: "France",
-	content: "This is an example message",
-	round_number: 0,
-	game_id: g.id
-)
+# m = Message.create(
```

Comment:
Is the example message intentionally commented out? At first I assumed yes, but the example news messages are left in a few lines below.